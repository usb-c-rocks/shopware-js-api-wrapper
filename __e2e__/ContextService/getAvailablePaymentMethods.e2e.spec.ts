import { getAvailablePaymentMethods } from "@shopware-js-api-wrapper";

describe("shopware-6-client - E2E - ContextService - getAvailablePaymentMethods", () => {
  it("should test paymentMethods response", async () => {
    const result = await getAvailablePaymentMethods();
    expect(result).toMatchSnapshot();
  });
});
