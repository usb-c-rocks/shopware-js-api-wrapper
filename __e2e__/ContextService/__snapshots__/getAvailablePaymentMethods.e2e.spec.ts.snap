// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shopware-6-client - E2E - ContextService - getAvailablePaymentMethods should test paymentMethods response 1`] = `
Array [
  Object {
    "_uniqueIdentifier": "d530765a9df64ae4ab881c0ac68cc7e4",
    "active": true,
    "afterOrderEnabled": false,
    "apiAlias": "payment_method",
    "availabilityRule": null,
    "availabilityRuleId": null,
    "createdAt": "2020-05-12T07:21:35+00:00",
    "customFields": null,
    "description": "Payment via PayPal - easy, fast and secure.",
    "extensions": Object {
      "foreignKeys": Object {
        "apiAlias": "array_struct",
      },
    },
    "id": "d530765a9df64ae4ab881c0ac68cc7e4",
    "media": null,
    "mediaId": null,
    "name": "PayPal",
    "pluginId": "abf4e95fc85f4e10b8750b5a53d9d397",
    "position": -100,
    "translated": Object {
      "customFields": Array [],
      "description": "Payment via PayPal - easy, fast and secure.",
      "name": "PayPal",
    },
    "translations": null,
    "updatedAt": "2020-05-12T07:23:32+00:00",
    "versionId": null,
  },
  Object {
    "_uniqueIdentifier": "c5bb531cebfb4cb38ca19d5233442bc5",
    "active": true,
    "afterOrderEnabled": true,
    "apiAlias": "payment_method",
    "availabilityRule": null,
    "availabilityRuleId": null,
    "createdAt": "2020-04-28T13:55:21+00:00",
    "customFields": null,
    "description": "Payment upon receipt of goods.",
    "extensions": Object {
      "foreignKeys": Object {
        "apiAlias": "array_struct",
      },
    },
    "id": "c5bb531cebfb4cb38ca19d5233442bc5",
    "media": null,
    "mediaId": null,
    "name": "Cash on delivery",
    "pluginId": null,
    "position": 1,
    "translated": Object {
      "customFields": Array [],
      "description": "Payment upon receipt of goods.",
      "name": "Cash on delivery",
    },
    "translations": null,
    "updatedAt": null,
    "versionId": null,
  },
  Object {
    "_uniqueIdentifier": "af6ab74164d7407981b02289b65aee34",
    "active": true,
    "afterOrderEnabled": true,
    "apiAlias": "payment_method",
    "availabilityRule": null,
    "availabilityRuleId": null,
    "createdAt": "2020-04-28T13:55:21+00:00",
    "customFields": null,
    "description": "Pay in advance and get your order afterwards",
    "extensions": Object {
      "foreignKeys": Object {
        "apiAlias": "array_struct",
      },
    },
    "id": "af6ab74164d7407981b02289b65aee34",
    "media": null,
    "mediaId": null,
    "name": "Paid in advance",
    "pluginId": null,
    "position": 2,
    "translated": Object {
      "customFields": Array [],
      "description": "Pay in advance and get your order afterwards",
      "name": "Paid in advance",
    },
    "translations": null,
    "updatedAt": null,
    "versionId": null,
  },
  Object {
    "_uniqueIdentifier": "3ab7b74f948a4fd2921cea2bf4a120a3",
    "active": true,
    "afterOrderEnabled": true,
    "apiAlias": "payment_method",
    "availabilityRule": null,
    "availabilityRuleId": null,
    "createdAt": "2020-04-28T13:55:21+00:00",
    "customFields": null,
    "description": "Payment by invoice. Shopware provides automatic invoicing for all customers on orders after the first. This is to avoid defaults on payment.",
    "extensions": Object {
      "foreignKeys": Object {
        "apiAlias": "array_struct",
      },
    },
    "id": "3ab7b74f948a4fd2921cea2bf4a120a3",
    "media": null,
    "mediaId": null,
    "name": "Invoice",
    "pluginId": null,
    "position": 5,
    "translated": Object {
      "customFields": Array [],
      "description": "Payment by invoice. Shopware provides automatic invoicing for all customers on orders after the first. This is to avoid defaults on payment.",
      "name": "Invoice",
    },
    "translations": null,
    "updatedAt": null,
    "versionId": null,
  },
]
`;
